.github/workflows/update-configuration.yml

```diff
@@ -0,0 +1,59 @@
+name: "Update Configuration"
+
+on:
+  workflow_dispatch:
+  push:
+
+jobs:
+  update:
+    name: "Update Configuration in manifest.json"
+    runs-on: ubuntu-latest
+    permissions: write-all
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup node
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20.10.0"
+
+      - name: Install deps and run configuration update
+        run: |
+          yarn install --immutable --immutable-cache --check-cache
+          yarn tsc --noCheck --project tsconfig.json
+
+      - name: Update manifest configuration using GitHub Script
+        uses: actions/github-script@v7
+        with:
+          script: |
+            (async () => {
+              const fs = await import('fs/promises');
+              const path = await import('path');
+
+              const { pluginSettingsSchema } = await import("${{ github.workspace }}/src/types/plugin-inputs.js");
+
+              const manifestPath = path.resolve("${{ github.workspace }}", './manifest.json');
+              const manifest = JSON.parse(await fs.readFile(manifestPath, 'utf8'));
+
+              const configuration = JSON.stringify(pluginSettingsSchema);
+              manifest["configuration"] = JSON.parse(configuration);
+
+              const updatedManifest = JSON.stringify(manifest, null, 2);
+              console.log('Updated manifest:', updatedManifest);
+              await fs.writeFile(manifestPath, updatedManifest);
+            })();
+
+      - name: Commit and Push generated types
+        run: |
+          git config --global user.name 'github-actions[bot]'
+          git config --global user.email 'github-actions[bot]@users.noreply.github.com'
```

Comment:
Auth as our bot using the github action in the previous step as we do in many other actions to get the auth token based on the `APP_ID` and `APP_PRIVATE_KEY`